code/modules/shuttle/super_cruise/shuttle_components/shuttle_docking.dm

```diff
@@ -174,6 +174,7 @@
 		my_port.dheight = shuttle_port.dheight
 		my_port.dwidth = shuttle_port.dwidth
 		my_port.hidden = shuttle_port.hidden
+		my_port.delete_after = TRUE
```

Comment:
The thing is, the dock already has delete_after since it gets it when its created.
I could land in the same position multiple times on the live server, the issue has something else involved. Could it be specifically custom shuttles being broken somehow?

Reply:
Honestly, lines 162 to 167 are redundant because my_port should be deleted after every take off, so there should be no situation where it'll need to be modified when landing in another custom location. Also, because custom shuttles can change in size now, the code for updating my_port's width, height, dwidth, and dheight should always be run, not just in an if statement. If you want I can go through this a bit more and clean it up some more, or I can just leave it like this if you don't want me messing with it too much.